src/pages/Home.tsx

```diff
@@ -0,0 +1,38 @@
+import { db as firebaseDB } from "../services/firebase-client";
+import { useEffect, useState } from "react";
+import { RegistrationForm } from '../components/RegistrationForm'
+
+interface User {
```

Comment:
if there is a `service.ts`, the typing would be inside that module
=> representation layer do not defines models